WeakAuras/RegionTypes/AuraBar.lua

```diff
@@ -404,8 +403,8 @@ local barPrototype = {
           end
 
           extraTexture:ClearAllPoints();
-          extraTexture:SetPoint(self.align1, self, self.align1, xOffset, yOffset);
-          extraTexture:SetPoint(self.align2, self, self.align2, xOffset, yOffset);
+          extraTexture:SetPoint(self.align1, self, self.align1, xOffset * Private.PixelMult, yOffset * Private.PixelMult);
```

Comment:
This can't be correct. Let's use these terms:
"uiscale-pixel" => a pixel in wow's coordinate system, unmodifed
"perfect pixel" => a pixel that is adjusted to be pixel-perfect

xOffset is derived from width (and progress)
* width is from GetRealSize()
* which is a uiscale pixel
Thus xOffset is in uiscale pixels.